hr_attendance_report_theoretical_time/tests/test_hr_attendance_report_theoretical_time.py

```diff
@@ -99,11 +99,16 @@ def setUpClass(cls):
         )
         # Force employee create_date for having auto-generated report entries
         cls.env.cr.execute(
-            "UPDATE hr_employee SET create_date = %s " "WHERE id in %s",
-            ("1946-12-23 12:00:00", (cls.employee_1.id, cls.employee_2.id)),
+            """
+            UPDATE hr_employee
+            SET create_date = '{}'
+            WHERE id IN ({}, {})
+        """.format("1946-12-23 12:00:00", cls.employee_1.id, cls.employee_2.id)
```

Comment:
```suggestion
       from odoo.tools import SQL

       cls.env.cr.execute(
            SQL(
                "UPDATE hr_employee SET create_date = %s WHERE id in (%s, %s)",
                "1946-12-23 12:00:00",
                cls.employee_1.id,
                cls.employee_2.id,
            )
        )
```

you can use new [SQL wrapper](https://www.odoo.com/documentation/18.0/developer/reference/backend/orm.html#odoo.tools.SQL) feature